pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -186,7 +207,11 @@ func (l *listUsersQuery) expand(
 	req *internalListUsersRequest,
 	foundUsersChan chan<- *openfgav1.User,
 ) error {
+	ctx, span := tracer.Start(ctx, "expand")
+	defer span.End()
+
 	if enteredCycle(req) {
+		span.SetAttributes(attribute.Bool("cycle_detected", true))
```

Comment:
**Question:** Similarly to here, do you think it's worth to add an attribute for `doesHavePossibleEdges` validations that return false?

Reply:
Yes I can add one.